CHANGELOG.md

```diff
@@ -51,6 +50,7 @@ in the naming of release branches.
   - `addrShelleyWitsL` to `addrShelleyTxWitsL`
   - `bootAddrShelleyWitsL` to `bootAddrShelleyTxWitsL`
   - `scriptShelleyWitsL` to `scriptShelleyTxWitsL`
+- Updated package metadata #3023
```

Comment:
I don't think this is needed, changelog usually only reflects code changes that can affect downstream users.

```suggestion
```